packages/react-components/react-alert/src/components/Alert/useAlertStyles.ts

```diff
@@ -60,9 +65,15 @@ const useIntentIconStyles = makeStyles({
  */
 export const useAlertStyles_unstable = (state: AlertState): AlertState => {
   const styles = useStyles();
+  const appearance = state.appearance || 'primary';
   const intentIconStyles = useIntentIconStyles();
 
-  state.root.className = mergeClasses(alertClassNames.root, styles.root, state.root.className);
+  state.root.className = mergeClasses(
+    alertClassNames.root,
+    styles.root,
+    appearance !== 'primary' && styles.invertedRoot,
```

Comment:
```suggestion
    appearance !== 'primary' && styles.inverted,
```

IMO we don't use "root" suffix for styles defined for "root" slot